### PR TITLE
Some small fixes to copy

### PR DESCRIPTION
**Pull request by beshaya**
Fixes: #150
Fixes: #151
Fixes: #152

### CODE REVIEW

File: src/components/calculator/ActivityRiskControls.tsx [hunk 1]
```diff
@@ -63,7 +63,7 @@ export const ActivityRiskControls: React.FunctionComponent<{
       />
       <SelectControl
         id="setting"
-        label="Setting"
+        label="Ventillation"
```
Comment (oremanj) [inline src/components/calculator/ActivityRiskControls.tsx:66]: Ventilation is spelled with one L

File: src/paper/5-activity-risk.ts [hunk 2]
```diff
@@ -74,7 +74,7 @@ Hopefully this puts the risk of indoor gatherings into perspective. To compare t
...
+**Masks**: We estimate that masks reduce your risk by 8x. This is if _both_ people are wearing masks. _Your_ mask decreases the risk to _you_ by about 2x. And _their_ mask decreases risk to _you_ by 4x. This assumes [a reasonably well-fitting surgical mask](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3306645/). For more protection, there are other types of masks you can wear (addressed in the [Q&A](/paper/13-q-and-a)). Bandanas, buffs, or other single-layer coverings provide significantly less protection than we estimate here (see [Research Sources](/paper/14-research-sources), so for simplicity we treat them as "no mask", even though we do believe they provide some benefit).
```
Comment (oremanj) [inline src/paper/5-activity-risk.ts:77]: I'm pretty sure the close paren is meant to be right after "Research Sources", not at the end of the sentence.

File: src/components/calculator/ActivityRiskControls.tsx (hunk 1 above)
Comment (oremanj) [inline src/components/calculator/ActivityRiskControls.tsx:66]:
```suggestion
        label="Ventilation"
```

File: src/paper/5-activity-risk.ts (hunk 2 above)
Comment (oremanj) [inline src/paper/5-activity-risk.ts:77]:
```suggestion
**Masks**: We estimate that masks reduce your risk by 8x. This is if _both_ people are wearing masks. _Your_ mask decreases the risk to _you_ by about 2x. And _their_ mask decreases risk to _you_ by 4x. This assumes [a reasonably well-fitting surgical mask](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3306645/). For more protection, there are other types of masks you can wear (addressed in the [Q&A](/paper/13-q-and-a)). Bandanas, buffs, or other single-layer coverings provide significantly less protection than we estimate here (see [Research Sources](/paper/14-research-sources)), so for simplicity we treat them as "no mask", even though we do believe they provide some benefit.
```

File: src/components/calculator/ActivityRiskControls.tsx (hunk 1 above)
Comment (beshaya) [reply src/components/calculator/ActivityRiskControls.tsx]: Done

File: src/paper/5-activity-risk.ts (hunk 2 above)
Comment (beshaya) [reply src/paper/5-activity-risk.ts]: Done

**Comment by catherio** [comment]
looks like josh has the baton here

**Pull request merged**